src/sentry/utils/encryption.py

```diff
@@ -36,7 +36,9 @@ def encrypt(self, value):
             return value
         value = smart_bytes(value)
         scheme = self.schemes[self.default_scheme]
-        return u"{}{}${}".format(MARKER, self.default_scheme, b64encode(scheme.encrypt(value)))
+        return u"{}{}${}".format(
+            MARKER, self.default_scheme, force_text(b64encode(scheme.encrypt(value)))
+        )
 
     def decrypt(self, value):
```

Comment:
After reading some more I'm pretty sure we need to return a `text_type` from here. Looking at `EncryptedJsonField.to_python`, it decodes then passes to the parent `to_python`. That method requires the value to be a `string_type`, which is only a unicode string in py3. If it's not it'll just return the bytes without loading into a dict.

I see similar issues in `EncryptedPickledObjectField` as well.

Reply:
I added docstrings to help indicate how these work now. They will always return strings now.